src/dnsmasq_interface.c

```diff
@@ -77,6 +77,7 @@ static union all_addr null_addrp = {{ 0 }};
 static enum reply_type force_next_DNS_reply = REPLY_UNKNOWN;
 static int last_regex_idx = -1;
 static struct ptr_record *pihole_ptr = NULL;
+#define HOSTNAME "Pi-hole hostname"
```

Comment:
Is this from the hostname PR or is this related to this PR?

Reply:
It is intended here, now that we need the string in two places, I made it a preprocessor define.

Reply:
Great.